code/modules/shuttle/shuttles/dropship.dm

```diff
@@ -81,6 +81,16 @@
 	. = ..()
 	control_doors("force-lock-launch", "all", force=TRUE, asynchronous = FALSE)
 
+	for(var/area/checked_area in shuttle_areas)
+		for(var/turf/checked_turf in checked_area)
```

Comment:
IIRC you don't need to check turfs, if mobs are on map they're also in area

Reply:
> The contents of areas are a little different. The turfs contained in the area are in the list along with any objs or mobs directly contained by those turfs.

according to the ref, yep!